hosting/nginx.prod.conf.hbs

```diff
@@ -60,7 +60,7 @@ http {
     add_header X-Frame-Options SAMEORIGIN always;
     add_header X-Content-Type-Options nosniff always;
     add_header X-XSS-Protection "1; mode=block" always;
-    add_header Content-Security-Policy ${csp_default}; ${csp_script}; ${csp_style}; ${csp_object}; ${csp_base_uri}; ${csp_connect}; ${csp_font}; ${csp_frame}; ${csp_img}; ${csp_manifest}; ${csp_media}; ${csp_worker};" always;
+    add_header Content-Security-Policy "${csp_default}; ${csp_script}; ${csp_style}; ${csp_object}; ${csp_base_uri}; ${csp_connect}; ${csp_font}; ${csp_frame}; ${csp_img}; ${csp_manifest}; ${csp_media}; ${csp_worker};" always;
```

Comment:
Can we add these to the dev config as well? There have been a few issues with CSP in the past, mainly because we don't test out this configuration before deploying to production. Or is there something blocking us from adding to dev?

Reply:
There's nothing blocking, the dev one just has to account for all the different URLs that we use locally, and it gets in your way